spec/features/admin_adds_new_video_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+feature 'Admin adds new video' do
+  scenario "Admin successfully adds a new video" do
```

Comment:
I would probably refactor this spec a little bit - to different methods. especially the expectations. Not a big deal though.